src/components/ConversionFormOverlay/index.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from "react";
+import { Upload, Select, Divider, message, Button } from "antd";
+import { UploadChangeParam } from "antd/lib/upload";
+import { ActionCreator } from "redux";
+
+import {
+    ClearSimFileDataAction,
+    LocalSimFile,
+} from "../../state/trajectory/types";
+import {
+    ResetDragOverViewerAction,
+    SetErrorAction,
+} from "../../state/viewer/types";
+import { Loading } from "../Icons"; // removed UploadFile icon
+import customRequest from "../LocalFileUpload/custom-request-upload";
+import { SetViewerStatusAction } from "../../state/viewer/types";
+import { DownArrow } from "../Icons";
+import type { UploadFile } from "antd/es/upload/interface";
+
+const { Dragger } = Upload;
+
+import styles from "./style.css";
+
+interface ConversionFormOverlayProps {
+    [key: string]: any;
+    isLoading: boolean;
+}
+
+const fileList: UploadFile[] = [
+    {
+        uid: "0",
+        name: "xxx.png",
+    },
+];
+
+const ConversionFormOverlay = ({
+    isLoading, // why doesnt this work when i try to write isLoading: true
+}: ConversionFormOverlayProps): JSX.Element | null => {
+    const [showTarget, setVisibility] = useState(true);
+    const [fileList, setFileList] = useState<UploadFile[]>([
+        {
+            uid: "-1",
+            name: "default.format",
+            status: "done",
+            url: "https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png",
+        },
+    ]);
+
+    // taken from ViewerOverlayTarget
+    const loadingOverlay = (
+        <div className={styles.container}>
+            <p className="loading-icon">{Loading}</p>
+            <p className="loading-text">Loading Simularium Model</p>
+        </div>
+    );
+
+    const conversionFormOverlay = (
+        <div className={styles.container}>
+            <h3 className={styles.title} style={{ fontSize: 30 }}>
+                Import a non-native file type
+            </h3>
+            <h3>
+                Convert and import a non-simularium file by providing the
+                following information
+            </h3>
+            <h3> Provide file information (required) </h3>
+            <h3 style={{ paddingBottom: 0 }}>Simulation Engine</h3>
```

Comment:
avoid using inline style as much as possible. use a className here and apply the style in the style sheet